src/httprx.js

```diff
@@ -4,6 +4,9 @@ const https = require('https');
 
 const request = httpClient => (options, data, isBinary) => new rx.Observable(obs => {
     const req = httpClient.request(options, resp => {
+        if (resp.statusCode > 399) {
```

Comment:
Shouldn't this be handled by consumers of the stream?

Reply:
Hmm, not sure. Angular http client, invokes error if response is not 2xx: https://angular.io/guide/http#error-handling. But then I found another Node library that puts it on the consumer to check for it. Your call...

I'm a little unclear on the difference between throwing an error and invoking the observable's error method. I went with this approach because you already had a catch in deploy.js, which is where the first came up -- we thought a deploy had succeeded but it hadn't.

I will read up on this some more.

Reply:
Actually there seem to be 3 ways of generating an error, and I think the code currently uses all there:

* throw new Error()
* Invoke error() method on the Observable object
* return rx.Observable.throw()

The nuances are unknown to me. I will read up on them.